airflow/api_fastapi/execution_api/datamodels/variable.py

```diff
@@ -32,7 +32,5 @@ class VariableResponse(BaseModel):
 class VariablePostBody(StrictBaseModel):
     """Request body schema for creating variables."""
 
-    model_config = ConfigDict(extra="forbid")
-
-    value: str | None = Field(serialization_alias="val")
+    value: str | None = Field(alias="val")
```

Comment:
This change will change it in the _generated.py, maybe we shouldn't do this

Reply:
Sadly this was required to work around https://github.com/zmievsa/cadwyn/issues/260 -- and I changed the client code to reflect this too.